src/main/java/io/jenkins/plugins/propelo/commons/plugins/Common.java

```diff
@@ -4,6 +4,8 @@
 
 public class Common {
     public static final String API_URL_PROD = "https://api.propelo.ai";
+    public static final String HARNESS_API_URL_PROD = "https://app.harness.io/sei/api";
+    public static final String HARNESS_COMPLIANCE_API_URL_PROD = "https://app3.harness.io/sei/api";
```

Comment:
Since we are going down this path... probably better to include the 3 Legacy Prod envs as options as well

Reply:
Added support for EU and Asia as well